cds-snc/notification-admin
# Use correct translation key and fix typo

# Summary | Résumé

- Fixes a small typo in some hint text
- Uses the correct translation key for the add recipients button

# Testing instructions:

- Create a draft and check the hint text in English
- On a template page, make sure the buttons have the right text in French and in English
